otter/test/convergence/test_gathering.py

```diff
@@ -111,7 +122,8 @@ def test_get_all_without_link_to_next_page(self):
         body = {'servers': self.servers}
         eff = get_all_server_details(batch_size=10)
         svcreq = resolve_retry_stubs(eff)
-        result = resolve_svcreq(svcreq, (fake_response, body), *self.req())
```

Comment:
Non-blocking - I think `self.req` is no longer called anywhere - can it be removed?